src/desktop/apps/about/test/client/router.test.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * decaffeinate suggestions:
+ * DS102: Remove unnecessary code created because of implicit returns
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+const _ = require("underscore")
+const benv = require("benv")
+const rewire = require("rewire")
+const sinon = require("sinon")
+const Backbone = require("backbone")
+
+describe("AboutRouter", function () {
```

Comment:
One meta comment: Is it possible to run decaffinate with a preference towards arrow functions? In day-to-day JS dev one pretty rarely sees `this`, and I worry that in that in the future `this` will be understood even less than it is now.

Reply:
I believe that it is. However, in our CoffeeScript almost every file I've looked at use `this` or variables that begin with `@`. One of the goals in decaffeinating is to enable us to leverage more modern tooling, some of which doesn't support `.coffee`, so modernizing the code wasn't a high priority.